bin/.travis/test.sh

```diff
@@ -48,70 +51,73 @@ if [ "$REUSE_VOLUME" = "0" ]; then
         printf "\nAs requested will also force update packages after create-project\n"
         docker run -ti --rm \
           -e SYMFONY_ENV \
+          -e APP_ENV \
           -e PHP_INI_ENV_memory_limit=3G \
           -v $(pwd)/volumes/ezplatform:/var/www \
           -v  $COMPOSER_HOME:/root/.composer \
-          ez_php:latest-node \
+          ez_php:latest-dev \
           bash -c "
           composer --version &&
           composer create-project --prefer-dist --no-progress --no-interaction --no-scripts ezsystems/ezplatform /var/www $EZ_VERSION &&
           composer update --prefer-dist --no-progress --no-interaction --with-all-dependencies"
     else
         docker run -ti --rm \
           -e SYMFONY_ENV \
+          -e APP_ENV \
+          -e PHP_INI_ENV_memory_limit=3G \
           -v $(pwd)/volumes/ezplatform:/var/www \
           -v  $COMPOSER_HOME:/root/.composer \
-          ez_php:latest-node \
+          ez_php:latest-dev \
           bash -c "
           composer --version &&
           composer create-project --prefer-dist --no-progress --no-interaction ezsystems/ezplatform /var/www $EZ_VERSION"
     fi
 fi
 
-printf "\nMake sure Node.js and Yarn are included in latest-node and latest-dev\n"
-docker -l error run -a stderr ez_php:latest-node node -e "process.versions.node"
+printf "\nMake sure Composer2 is available in latest\n"
+docker -l error run -a stderr ez_php:latest bash -c "composer2 --version"
+
+printf "\nMake sure Node.js and Yarn are included in latest-dev\n"
 docker -l error run -a stderr ez_php:latest-dev node -e "process.versions.node"
-docker -l error run -a stderr ez_php:latest-node bash -c "yarn -v"
 docker -l error run -a stderr ez_php:latest-dev bash -c "yarn -v"
 
-printf "\nMinimal testing on ez_php:latest for use with ez user\n"
-docker run -ti --rm \
-  -v $(pwd)/volumes/ezplatform:/var/www \
-  -v $(pwd)/bin/.travis/testSymfonyRequirements.php:/var/www/testSymfonyRequirements.php \
-  ez_php:latest \
-  bash -c "php testSymfonyRequirements.php"
-
-
-printf "\nMinimal testing on ez_php:latest-dev for use with ez user\n"
-docker run -ti --rm \
-  -v $(pwd)/volumes/ezplatform:/var/www \
-  -v $(pwd)/bin/.travis/testSymfonyRequirements.php:/var/www/testSymfonyRequirements.php \
-  ez_php:latest-dev \
-  bash -c "php testSymfonyRequirements.php"
-
+if [ "$EZ_VERSION" = "^2.5" ]; then
```

Comment:
This might fail easily.
Is there some equivalent for 3.0 to test requirements? If not we can consider dropping it.